src/scripts/main.js

```diff
@@ -1,3 +1,8 @@
 'use strict';
 
-// write code here
+const spider = document.querySelector('.spider');
```

Comment:
it's a good practice to add ref/el suffix to variables with a reference to a node element
```suggestion
const spiderRef = document.querySelector('.spider');
```